config/vault.go

```diff
@@ -233,18 +246,26 @@ func (c *VaultConfig) Finalize() {
 	}
 	c.SSL.Finalize()
 
+	// Order of precedence
+	// 1. `vault_agent_token_file` configuration value
+	// 2. `VAULT_TOKEN` environment variable
+	// 3. `token` configuration value
+	// 4. `~/.vault-token` file
```

Comment:
I don't think we should rely on this file or maybe worth further discussion.

Reply:
Agreed - I couldn't remember this file being part of the typical Vault defaults but wasn't sure whether to remove it.

Reply:
I just checked the code and I am pretty sure the token helper, which uses the .vault-token file, is only used by the CLI and not the default API config.  I would leave it out since it seems to only be used with interactive users.